cmd/snap/cmd_run_test.go

```diff
@@ -658,11 +658,11 @@ func (s *SnapSuite) TestAntialiasHappy(c *check.C) {
 	app, outArgs = snaprun.Antialias("alias", inArgs)
 	c.Check(app, check.Equals, "an-app")
 	c.Check(outArgs, check.DeepEquals, []string{
-		"99", // COMP_TYPE (no change)
-		"99", // COMP_KEY (no change)
-		"11", // COMP_POINT (+1 because "an-app" is one longer than "alias")
-		"2",  // COMP_CWORD (no change)
-		" ",  // COMP_WORDBREAKS (no change)
+		"99",                    // COMP_TYPE (no change)
```

Comment:
Go 1.11 `gofmt` did this.

Reply:
Hm, my go fmt 1.10 will undo this again. Do you mind if I push a change until most of use have switched to 1.11?

Reply:
Go ahead.